### DISCUSSION TITLE
API Redesign Released in `v2023.06.0`

### DISCUSSION THREAD

**Discussion by philipc2** [UXARRAY/uxarray#351] [Announcements]
The ongoing overhaul to UXarray's API design has officially been released as of `v2023.06.0`

A detailed overview of the changes can be found in the following places
* [`v2023.06.0` Release Page](https://github.com/UXARRAY/uxarray/releases/tag/v2023.06.0)
* [GitHub Pull Request #216 - Redesign](https://github.com/UXARRAY/uxarray/pull/216) 
* [GitHub Discussion - UXarray Redesign Thoughts and Options](https://github.com/UXARRAY/uxarray/discussions/185)
* [Updated Usage Examples](https://uxarray.readthedocs.io/en/latest/examples.html) 
* [Updated User API](https://uxarray.readthedocs.io/en/latest/user_api/index.html)

**Comment by erogluorhan** [comment]
Thank you to all our community members who were involved in brainstorming, discussion, implementation, and review for this effort!
